### PR TITLE
Update afe link on code.org/about/supporters/amazon

### PR DESCRIPTION
**Pull request by kelbyhawn**
Updates link on https://code.org/about/supporters/amazon.

## Links
Gdoc comment: [here](https://docs.google.com/document/d/1Sw3r0zZnz-8ZTFEDdSoyupErnTGAWhrxxL_OT1AGAac/edit?disco=AAABVt1vJ6c)

## Testing story
Tested locally

### CODE REVIEW

**Review by hannahbergam — COMMENTED** [review]
I've never seen a url like this- what does the google.com/url? piece add?

File: pegasus/sites.v3/code.org/public/about/supporters/amazon.haml [hunk 1]
```diff
@@ -46,4 +46,4 @@ theme: responsive
 
 %p Amazon's commitment to innovation remains steadfast as it develops programs that equip students for future careers. Harnessing Amazon's culture of invention and Code.org's expertise, this collaboration prepares students for future careers in computer science and STEM, nurturing the next generation of innovators.
 
-%p More information on Amazon Future Engineer is <a href="https://www.amazonfutureengineer.com/" target="_blank" rel="nooopener nooreferrer" class="has-external-link">available here</a>.
+%p More information on Amazon Future Engineer is <a href="https://www.google.com/url?q=https://www.amazonfutureengineer.com/?utm_campaign%3DCode.org%26utm_medium%3DWebsite%2520%26utm_source%3DUS%26utm_content%3DSupporter%2520Page%26utm_term%3D2024&sa=D&source=docs&ust=1727304007862733&usg=AOvVaw3VsvLIwZPioIbLKgbeDHg8" target="_blank" rel="nooopener nooreferrer" class="has-external-link">available here</a>.
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/public/about/supporters/amazon.haml]: I think this copied with some characters that should not be in here

Comment (dmcavoy) [reply pegasus/sites.v3/code.org/public/about/supporters/amazon.haml]: https://www.amazonfutureengineer.com/?utm_campaign=Code.org&utm_medium=Website%20&utm_source=US&utm_content=Supporter%20Page&utm_term=2024

**Comment by kelbyhawn** [comment]
> I've never seen a url like this- what does the google.com/url? piece add?

It's for tracking! According to Google AI: "A UTM link, or Urchin Tracking Module link, is a URL with a UTM code attached to it that tracks the performance of a link or campaign."

File: pegasus/sites.v3/code.org/public/about/supporters/amazon.haml (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/about/supporters/amazon.haml]: Ah you're right! Updated 👍

**Review by hannahbergam — APPROVED** [review]
Ahh looks like the extra chars i was asking about got removed! LGTM!

**Pull request merged**